whistl/classifier.py

```diff
@@ -30,7 +30,8 @@ def compute_irm_penalty(loss, dummy_w):
 
 
 def train_with_irm(classifier, map_file, train_dirs, tune_dirs, gene_file,
-                   num_epochs, loss_scaling_factor, label_to_encoding, device, logger=None):
+                   num_epochs, loss_scaling_factor, label_to_encoding, device, logger=None,
+                   save_file=None, burn_in_epochs=100):
     '''Train the provided classifier using invariant risk minimization
     IRM looks for features in the data that are invariant between different environments, as
```

Comment:
Just out of curiosity, what do you mean by invariant here?

Reply:
I may have already explained this [earlier](https://github.com/greenelab/whistl/pull/13#discussion_r350779490), but they are invariant in the sense that the model doesn't have more or less error in different environments. For more information on the details of how that is accomplished, see https://arxiv.org/abs/1907.02893